src/content/changelogs/access.yaml

```
@@ -5,6 +5,10 @@ productLink: "/cloudflare-one/applications/configure-apps/"
 productArea: Cloudflare One
 productAreaLink: /cloudflare-one/changelog/
 entries:
+  - publish_date: "2025-02-05"
```

```suggestion
  - publish_date: "2025-02-02"
```
We should move this entry to `/changelogs/gateway.yaml` since it mostly affects Gateway products.